locale/en/invitation.po

```diff
@@ -134,16 +134,10 @@ msgid "userInvitation.searchUser.nextButtonLabel"
 msgstr "Search User"
 
 msgid "userInvitation.searchUser.stepDescription"
-msgstr "Search for the user using their email address, username or ORCID ID. Enter at least one details to get started. If the user does not exist, you can invite them to take up roles and be a part of your journal. If the user already exist in the system, you can view user information and invite to take a additional roles."
+msgstr "If the user does not exist, you can invite them to take on roles and become part of your journal. If the user already exists in the system, you can view their information and invite them to take on additional roles."
 
-msgid "userInvitation.emailField.description"
-msgstr "e.g. aeinstein@example.com"
-
-msgid "userInvitation.usernameField.description"
-msgstr "e.g. mickeymouse"
-
-msgid "userInvitation.orcidField.description"
-msgstr "e.g. 0000-0000-0000-0000"
+msgid "userInvitation.searchField.description"
+msgstr "e.g. aeinstein@example.com or aeinstein or 0000-0000-0000-0000"
```

Comment:
I wonder if we should use `0000-0002-1825-0097` as the ORCID example, one provided by ORCID as an official example that includes a valid checksum and is a valid ORCID (see: https://support.orcid.org/hc/en-us/articles/360006897674-Structure-of-the-ORCID-Identifier)